tesler-core/src/main/java/io/tesler/core/metahotreload/conf/properties/MetaConfigurationProperties.java

```diff
@@ -0,0 +1,51 @@
+/*-
+ * #%L
+ * IO Tesler - Core
+ * %%
+ * Copyright (C) 2018 - 2019 Tesler Contributors
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+package io.tesler.core.metahotreload.conf.properties;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.validation.annotation.Validated;
+
+@Getter
+@Setter
+@Validated
+@ConfigurationProperties(prefix = "tesler.meta")
+public class MetaConfigurationProperties {
+
+	private boolean enabled = false;
+
+	@NotNull(message = "Ant path to meta files directory. Supports file: or classpath: prefix. "
+			+ "Example of usage is: applicationContext.getResources(directory + widgetPath)")
+	private String directory = "classpath*:db/migration/liquibase/data/latest";
+
+	@NotBlank(message = "ant path to widget files from MetaConfigurationProperties.directory")
```

Comment:
I guess we need to simplify the message, because i dont think that most of developers knows about "Ant paths". Or use javadoc to link the descriptions about ant paths.

Reply:
fixed